services/scrutinizer/scrutinizer-coverage.service.js

```diff
@@ -0,0 +1,157 @@
+'use strict'
+
+const Joi = require('joi')
+const { colorScale } = require('../color-formatters')
+const ScrutinizerBase = require('./scrutinizer-base')
+
+const schema = Joi.object({
+  default_branch: Joi.string().required(),
+  applications: Joi.object()
+    .pattern(
+      /^/,
+      Joi.object({
+        index: Joi.object({
+          _embedded: Joi.object({
+            project: Joi.object({
+              metric_values: Joi.object({
```

Comment:
Have to wonder if they ever intended for people to read these…

Reply:
🤣